src/components/NostoProduct/store.ts

```diff
@@ -121,3 +139,8 @@ export function injectStore(element: HTMLElement, cb: Callback) {
 }
 
 export type Store = ReturnType<typeof createStore>
+
+function getSkuValue<T extends keyof Sku>(skus: Sku[], key: T) {
```

Comment:
This method looks a bit complicated

Reply:
what kind of improvements do you propose?

Reply:
Please [refer](https://github.com/Nosto/web-components/pull/154/files#r2065635959)